### PR TITLE
Add calendar keyboard shortcuts

### PR DESCRIPTION
**Pull request by JanGuillermo**
I wanted to increase productivity in Tutanota's Calendar feature by adding keyboard shortcuts.

**Planned**
- Add translations for all languages

**Implemented**
- Added a keyboard shortcut for all pages
  - Press the key `O` to switch to the calendar view.
- Added keyboard shortcuts on the calendar feature
  - **Navigation**
    - Press the key `1` to switch to the week view.
    - Press the key `2` to switch to the month view.
    - Press the key `3` to switch to the agenda view.
    - Press the key `T` to view today.
    - Press the key `J` to view the next period.
    - Press the key `K` to view the previous period.
  - **Actions**
    - Press the key `N` to create a new calendar event.
- Added translations for English (so far)

### CODE REVIEW

File: src/calendar/CalendarView.js [hunk 1]
```diff
@@ -272,11 +276,96 @@ export class CalendarView implements CurrentView {
...
+				help: "createEvent_action"
+			}
+		]
+
+		this.oncreate = () => keyManager.registerShortcuts(shortcuts)
+		this.onbeforeremove = () => keyManager.unregisterShortcuts(shortcuts)
+	}
+
+	_viewPeriod(next) {
+		let newDate = new Date(this.selectedDate().getTime())
```
Comment (charlag) [inline src/calendar/CalendarView.js]: This duplicates code from `onChangeWeek` and the like, I think we should extract it

Comment (JanGuillermo) [reply src/calendar/CalendarView.js]: I used the `_viewPeriod()` in `onChangeWeek` as well as `onChangeMonth`

**Comment by charlag** [comment]
Ah, I forgot one thing. I'm not sure we need day view. Right now it does not work with desktop properly and we don't have any way to access it besides URL hijacking. I don't think day view makes too much sense on desktop and I would remove it from shortcuts too.

**Comment by charlag** [comment]
Works great, thanks a lot!

**Comment by charlag** [comment]
New calendar keys are missing from Phrase:
`calendarView_action`
`viewNextPeriod_action`
`viewPrevPeriod_action`
`viewToday_action`

we should not use `createEvent_action` because we already have `newEvent_action`
we should not use `createCalendar_action` because we already have `addCalendar_action`

`switchAgendaView_action`, `switchDayView_action`, `switchMonthView_action` also seem to be redundant because we can just use translations from sidebar

**Comment by charlag** [comment]
Maybe someone with better German than mine can do this

**Comment by charlag** [comment]
Shortcuts are at least twice in the list and sometimes even more but that's probably another issue

**Pull request merged**